test/e2e/run_test.go

```diff
@@ -61,6 +63,77 @@ var _ = Describe("Podman run", func() {
 		Expect(match).Should(BeTrue())
 	})
 
+	It("podman run selinux disable test", func() {
+		selinux := podmanTest.SystemExec("ls", []string{"/usr/sbin/selinuxenabled"})
+		selinux.WaitWithDefaultTimeout()
+		if selinux.ExitCode() != 0 {
+			Skip("SELinux not enabled")
+		}
+		session := podmanTest.Podman([]string{"run", "-it", "--security-opt", "label=disable", ALPINE, "cat", "/proc/self/attr/current"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		match, _ := session.GrepString("unconfined_t")
+		Expect(match).Should(BeTrue())
+	})
+
+	It("podman run selinux type check test", func() {
+		selinux := podmanTest.SystemExec("ls", []string{"/usr/sbin/selinuxenabled"})
+		selinux.WaitWithDefaultTimeout()
+		if selinux.ExitCode() != 0 {
+			Skip("SELinux not enabled")
+		}
+		session := podmanTest.Podman([]string{"run", "-it", ALPINE, "cat", "/proc/self/attr/current"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		match1, _ := session.GrepString("container_t")
+		match2, _ := session.GrepString("svirt_lxc_net_t")
+		Expect(match1 || match2).Should(BeTrue())
+	})
+
+	It("podman run selinux type setup test", func() {
+		selinux := podmanTest.SystemExec("ls", []string{"/usr/sbin/selinuxenabled"})
+		selinux.WaitWithDefaultTimeout()
+		if selinux.ExitCode() != 0 {
+			Skip("SELinux not enabled")
+		}
+		session := podmanTest.Podman([]string{"run", "-it", "--security-opt", "label=type:spc_t", ALPINE, "cat", "/proc/self/attr/current"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		match, _ := session.GrepString("spc_t")
+		Expect(match).Should(BeTrue())
+	})
+
+	It("podman run seccomp undefine test", func() {
+		selinux := podmanTest.SystemExec("ls", []string{"/usr/sbin/selinuxenabled"})
+		selinux.WaitWithDefaultTimeout()
+		if selinux.ExitCode() != 0 {
+			Skip("SELinux not enabled")
+		}
+		session := podmanTest.Podman([]string{"run", "-it", "--security-opt", "seccomp=unconfined", ALPINE, "echo", "hello"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		match, _ := session.GrepString("hello")
+		Expect(match).Should(BeTrue())
+	})
+
+	It("podman run seccomp test", func() {
+		selinux := podmanTest.SystemExec("ls", []string{"/usr/sbin/selinuxenabled"})
+		selinux.WaitWithDefaultTimeout()
+		if selinux.ExitCode() != 0 {
+			Skip("SELinux not enabled")
+		}
+
+		jsonFile := filepath.Join(podmanTest.TempDir, "seccomp.json")
+		in := []byte(`{"defaultAction":"SCMP_ACT_ALLOW","syscalls":[{"name":"getcwd","action":"SCMP_ACT_ERRNO"}]}`)
+		WriteJsonFile(in, jsonFile)
```

Comment:
Should you error out here if WriteJsonFile fails?

Reply:
Yes, Thanks a lot for reminding. Will skip the tests if write seccomp.json failed.